buildSrc/src/main/kotlin/org/cqfn/save/buildutils/SpringBootConfiguration.kt

```diff
@@ -83,6 +80,12 @@ fun Project.configureSpringBoot(withSpringDataJpa: Boolean = false) {
         }
     }
 
+    tasks.withType<BootRun>().configureEach {
+        if (Os.isFamily(Os.FAMILY_MAC)) {
+            environment["SPRING_PROFILES_ACTIVE"] = "${environment["SPRING_PROFILES_ACTIVE"] ?: "dev"},mac"
```

Comment:
Does `environment` here refer to JavaExec's environment? Than how can `environment["SPRING_PROFILES_ACTIVE"]` be not null in the right hand side part of assignment?

Reply:
Makes sense, my bad, fixing it